docs/source/pillars/computing_technology.md

```diff
@@ -107,6 +107,13 @@ _The ability to run analyses requiring more compute resources, or more specialis
     For example, when using physical compute resources all sensitive data must be securely wiped before another user is given access to that same node.
     In a cloud hosted TRE virtual machines should be destroyed and recreated.
   - Mandatory
+* - A TRE could integrate with large-scale data analytics tools
```

Comment:
```suggestion
* - A TRE could integrate with large-scale data analytics tools.
```

Reply:
Not sure if "data analytics" is a compound adjective here. "data-analytics tools", maybe not because data and analytics are nouns?